src/OneExplorer/ConfigObject.ts

```diff
@@ -19,6 +19,191 @@ import * as ini from 'ini';
 import * as path from 'path';
 import * as vscode from 'vscode';
 
+/**
+ * 'Artifact' means includes
```

Comment:
Is it a kind of group unit? logically? not physically?

Reply:
Just a logical one :-D